GVFS/GVFS.Common/Maintenance/LooseObjectsStep.cs

```diff
@@ -51,15 +51,8 @@ public int CountLooseObjects()
             return count;
         }
 
-        /// <summary>
-        /// Writes loose object Ids to streamWriter
-        /// </summary>
-        /// <param name="streamWriter">Writer to which SHAs are written</param>
-        /// <returns>The number of loose objects SHAs written to the stream</returns>
-        public int WriteLooseObjectIds(StreamWriter streamWriter)
+        public IEnumerable<string> LooseObjectsBatch(int count)

```

Comment:
If this is easy to add a unit test for (I think it should be), I would.

Reply:
`WriteLooseObjectIds()` is covered by unit tests, so this method is covered by proxy.

`batchSize` _is_ better, thanks!